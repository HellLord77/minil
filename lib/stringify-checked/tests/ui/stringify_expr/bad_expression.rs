use stringify_checked::stringify_expr;

fn main() {
    stringify_expr!(match Some(0) {});
}
