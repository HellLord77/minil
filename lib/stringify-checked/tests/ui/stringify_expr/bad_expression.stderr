error[E0004]: non-exhaustive patterns: `None` and `Some(_)` not covered
 --> tests/ui/stringify_expr/bad_expression.rs:4:27
  |
4 |     stringify_expr!(match Some(0) {});
  |                           ^^^^^^^ patterns `None` and `Some(_)` not covered
  |
note: `Option<i32>` defined here
 --> $RUST/core/src/option.rs
  |
  | pub enum Option<T> {
  | ^^^^^^^^^^^^^^^^^^
...
  |     None,
  |     ---- not covered
...
  |     Some(#[stable(feature = "rust1", since = "1.0.0")] T),
  |     ---- not covered
  = note: the matched value is of type `Option<i32>`
help: ensure that all possible cases are being handled by adding a match arm with a wildcard pattern, a match arm with multiple or-patterns as shown, or multiple match arms
  |
4   ~     stringify_expr!(match Some(0) {
5   +         None | Some(_) => todo!(),
6   ~     });
    |
